lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -175,8 +218,8 @@ def local_client(app: FastAPI):
         yield c
 
 
-@pytest.fixture(scope="function", autouse=True)
-def dependency_overrides(app: FastAPI, db_session: Session, s3_client: S3Client) -> None:
+@pytest.fixture(scope="function")
+def dependency_overrides_fakes(app: FastAPI, db_session: Session, fake_s3_client: S3Client, fake_s3fs: S3FileSystem) -> None:
```

Comment:
i think this method is duplicated

Reply:
There should be one for fakes and another one for the real services, but I may have not pushed that change :( I'll make sure it's correct in the next push 👍